layouts/partials/home/hero.html

```diff
@@ -17,7 +17,7 @@
       <br />
 
       <div class="buttons is-centered">
-        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/latest/">
+        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/latest/" data-proofer-ignore>
```

Comment:
Why is this tag necessary? I.e., why can't the checker check `/docs/latest`?

Reply:
The link checker isn't run with the netlify_cli--it just builds the site, then checks the links in the public folder, so it's not aware of any netlify redirects (like latest).

Reply:
Hmm, it should be run with the redirects in effect.

Reply:
I agree, but that may be a different tool.